tests/stress/README.md

```diff
@@ -0,0 +1,126 @@
+# Stress Testing
+
+## Intro
+To stress test csvcubed, we use a tool called JMeter to run two separate test plans for the build command and inspect command respectively.  
+The test plans are run consecutively via a bash script and follow broadly the same structure:
+
+- Start running a performance monitor in the background
+- Generate a "maximally complex" CSV file with the number of rows the user has given as a parameter to the bash script.
+  1. This is a preprocess which is a groovy script which in turn runs a python script because JMeter will not execute `.py` files.
+      - The inspect command test then runs the build command on this CSV file so that the json-ld output generated by the build command can be used as the input to the inspect command.
+- Run the test's designated command 5 times using the results of the preprocess step above as inputs.
+  1. The performance monitor records CPU and Memory usage throughout the duration of the test's 5 runs.
+- A postprocess removes any unwanted resultant files.
+  1. Also a groovy script which in turn runs a python script.
+- Steps 2 - 4 are repeated for the second command
+- The performance monitor is terminated.
+- Finally, metrics files are cleaned up and placed into relevant folders.
+
+## Installation Guide
+
+### From Bash Script
+  #!/bin/bash
```

Comment:
You should put this in back ticks (`) to ensure it's easy to copy and paste and isn't subject to formatting. Also can you make it clear to the user that this script will only work on OSX? It won't work in any other environment which might confuse someone.